crates/cli_testing_examples/expects/expects.roc

```diff
@@ -0,0 +1,17 @@
+app "expects"
+    packages { pf: "zig-platform/main.roc" }
+    imports []
+    provides [main] to pf
+
+expect
+    a = 1
+    b = 2
+
+    a == b
+
+main =
+    x = 42
+    expect x != x
```

Comment:
Not a blocker, but would be nice to have multiple `expect`s and also multiple named idents per `expect`, just to make sure those cases are all working. 😄